src/components/partner/partner.resolver.ts

```diff
@@ -93,6 +98,29 @@ export class PartnerResolver {
     );
   }
 
+  @ResolveField(() => SecuredLanguages)
+  async languagesOfConsulting(
+    @Parent() partner: Partner,
+    @Loader(LanguageLoader) loader: LoaderOf<LanguageLoader>,
+  ): Promise<SecuredLanguages> {
+    const languages = (
+      await loader.loadMany(partner.languagesOfConsulting.value)
```

Comment:
The value passed to the loadMany here gives:

![Screenshot 2023-09-27 at 8 42 13 AM](https://github.com/SeedCompany/cord-api-v3/assets/138262921/2f64fde6-e31c-48b8-b8b9-6b8068b01e84)